frame/staking/src/lib.rs

```diff
@@ -1480,6 +1478,30 @@ decl_module! {
 	}
 }
 
+impl<T: Trait> MigrateAccount<T::AccountId> for Module<T> {
+	fn migrate_account(a: &T::AccountId) {
+		if let Some(controller) = Bonded::<T>::migrate_key_from_blake(a) {
+			Ledger::<T>::migrate_key_from_blake(controller);
+			Payee::<T>::migrate_key_from_blake(a);
+			Validators::<T>::migrate_key_from_blake(a);
+			Nominators::<T>::migrate_key_from_blake(a);
+			SlashingSpans::<T>::migrate_key_from_blake(a);
+		}
+	}
+}
```

Comment:
Note for people using this as inspiration for migration:
This migration will likely be incorrect, as the `Session` account migration depends on the stash account migration to happen before the controller account migration which means that the order of account migration cannot be arbitrary (and thus just enumerating them is not enough to guarantee a successful migration).
See [here](https://github.com/hicommonwealth/substrate/pull/17/files#diff-32ce5961608780655bf3e00b2cc07d59R2222) for an order independent implementation.